Ball.java

```diff
@@ -0,0 +1,12 @@
+public class Ball {
+int number;
+Colors color;
+public Ball(int number, Colors color){
+   this.number = number;
+   this.color = color;
+    }
+
+    public String toString(){
+    return "Ball" + "color" + number;

```

Comment:
The `toString` method should return a string that includes the actual color of the ball. Replace `'color'` with `color` to use the `Colors` enum value, and consider adding a space or other separator between 'Ball', the color, and the number for better readability.